src/celeritas/em/data/WentzelData.hh

```diff
@@ -24,11 +24,12 @@ struct WentzelIds
     ActionId action;
     ParticleId electron;
     ParticleId positron;
-    ParticleId proton;
+    // ParticleId proton; // TODO: enable when protons are supported
```

Comment:
I think you can just leave this as-is, since emitting ions is "optional" behavior